tanagra-aou-utils/apiContext.ts

```diff
@@ -0,0 +1,355 @@
+import React from "react";
```

Comment:
This file is a current hack to all for the passing of the bearer token until load balancer issue is fixed. This file is copied into the test deployment to override the current passing of the bearer token